components/org.wso2.carbon.identity.oauth.uma.resource.endpoint/src/main/java/org/wso2/carbon/identity/oauth/uma/resource/endpoint/impl/ResourceRegistrationApiServiceImpl.java

```diff
@@ -103,30 +124,34 @@ public Response registerResource(ResourceDetailsDTO requestedResource, MessageCo
     @Override
     public Response getResource(String resourceId, MessageContext context) {
 
-        Response response = null;
-
         try {
-            if (isResourceId(resourceId)) {
-                try {
+            if (isValidToken(context)) {
+
+                if (log.isDebugEnabled()) {
+                    log.debug("Access token is valid token and all required context information " +
+                            "are existing in the token.");
+                }
+
+                if (isResourceIdValid(resourceId)) {
+
+                    if (log.isDebugEnabled()) {
+                        log.debug("Valid resourceId.");
```

Comment:
Add more context information. Without any information, it is hard to track for what request this log is printed. Do this for all the debug logs.